kafka-rest/src/test/java/io/confluent/kafkarest/integration/ClusterTestHarness.java

```diff
@@ -319,7 +326,7 @@ public void tearDown() throws Exception {
   }
 
   protected Invocation.Builder request(String path, String templateName, Object templateValue,
-      Map<String, String> queryParams) {
+    Map<String, String> queryParams) {
```

Comment:
nit: The new indentation looks wrong - I'd also suggest putting each of the ctor parameters on its own line.